### DISCUSSION TITLE
xattr package on linux

### DISCUSSION THREAD

**Discussion by doc-sheet** [python-poetry/poetry#8735] [Q&A]
Hi!

According to [pyproject](https://github.com/python-poetry/poetry/blob/0578ed81787449d79430360de143d916006e028b/pyproject.toml#L60) xattr should only be installed on windows.

But I have it on my linux machine
```
poetry 1.7.1 Python dependency management and packaging made easy.
└── xattr >=0.10.0,<0.11.0
    └── cffi >=1.0 
        └── pycparser * 
```

Did I miss something?

**Comment by radoering** [comment]
You probably did not use one of the [recommended ways](https://python-poetry.org/docs/#installation) to install Poetry. I suppose you either did not install Poetry into a dedicated virtual environment or you installed a modified version of Poetry from a source that is not maintained by the Poetry team.

**Comment by doc-sheet** [comment]
My bad. python3-pyxattr is a system wide package.
poetry just `feels` it inside venv.
